messages/lightning.preview.site.md

```diff
@@ -1,38 +1,35 @@
 # summary
 
-Preview Experience Cloud Sites.
+Preview an Experience Cloud site locally and in real-time, without deploying it.
 
 # description
 
-In dev preview mode, you can edit local files and see these changes to your Lightning Web Components (LWC) within your site:
+Enable Lightning Preview to see local changes to your site in a real-time preview that you don't have to deploy or manually refresh. To let you quickly iterate on your Lightning web components (LWCs) and pages, your site preview automatically refreshes when Lightning Preview detects source code changes.
+
+When you edit local files with Lightning Preview enabled, your LWCs reflect the following changes in your org. To apply any other local changes not listed here, you have to deploy them to your org using the `project deploy start` command.
 
 - Basic HTML and CSS edits
-- Importing new CSS
-- Javascript edits in-service component library
-- Javascript method changes in the LWC component that don't alter its public API.
+- Importing new CSS-only LWCs
+- JS edits in-service component library
+- JS method changes that don't affect the LWC's public API
```

Comment:
I need to test and confirm this actual list of whats supported for sites, but I can get back to you with the official list

Reply:
Ok the new list should be:
- Basic HTML and CSS edits
- Javascript edits in-service component library
- Javascript method changes that don't affect the LWC's public API
- Importing new custom LWCs
- Importing LWCs that already exist on the current page (not sure how to best phrase this)